docs/src/source-types.md

```diff
@@ -20,6 +20,9 @@ Cassandra:
   # * when true: the connection is dropped.
   # * when false: the connection will wait until a connection can be made within the limit.
   hard_connection_limit: false
+ 
+  #  Timeout in seconds after which to terminate an idle connection. This field is optional, if not provided, idle connections will never be terminated.
+  # timeout: 60
```

Comment:
Hmmm, all the other optional fields we have provided uncommented as it makes it more obvious that it is the field and not just a comment. see .e.g. tls field on RedisSinkSingle

However I think in this situation you have made the right call as we want people to use the default value of no timeout.